lib/QubitXmlImport.class.php

```diff
@@ -942,23 +942,25 @@ public static function runFilters(&$node, $filterParam)
   {
     foreach ($filterParam as $filters)
     {
-      foreach ($filters as $eadTag => $object)
+      foreach ($filters as $eadTag => $objects)
```

Comment:
I'd call it `$tag`, as we could use this function in other imports in the future.

Reply:
True - good point.  Yes my intent was to be able to use these functions for whatever import filtering we may need to do in the future.